File: Assignment47&50/program15.java
//write java program which accept number from user and return difference between summation of even digits and summation of odd digits

import java.lang.*;
import java.util.*;

class Digit
{
    public int CountDiff(int iNo)
    {
        int idigit = 0, iSum1 = 0,iSum2 = 0;
        int i = 0;  

        if(iNo == -iNo)
        {
            iNo = -iNo;
        }
        while(iNo != 0)
    {
        idigit = iNo % 10;
        if(idigit % 2 == 0)
        {
            iSum1 = iSum1+idigit;
        }
        else
        {
            iSum2 = iSum2+idigit;
        }
        iNo = iNo/10;
    }
        return iSum1 - iSum2;
    }
   
}


class program15
{
    public static void main(String arg[])
    {
        
        int iNo = 0;
        
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the Number..");
        iNo = sobj.nextInt();

        Digit dobj = new Digit();
       int iRet = dobj.CountDiff(iNo);
        System.out.println("Difference between Summation of Even number and odd number : "+iRet);
        
    }
}

File: Assignment5&7/program15.c
//write a program which accept number from user and count frequency of 4 in it..

#include<stdio.h>

int CountFour(int iNo)
{
    int iCount = 0;

    while(iNo>0)
    {
        if(iNo % 10 == 4)
        {
            iCount++;
        }
        iNo = iNo/10;

    }
    return iCount;
}

int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter Number\n");
    scanf("%d",&iValue);

    iRet = CountFour(iValue);
    printf("%d\n",iRet);
    return 0;
}